src/core/sprites/Sprite.js

```diff
@@ -63,9 +63,9 @@ function Sprite(texture)
      * @member {number}
      * @default 0xFFFFFF
      */
+    this.tint = 0xFFFFFF;
     this._tint = null;
     this._tintRGB = null;
-    this.tint = 0xFFFFFF;
```

Comment:
This will cause a bug, the private versions need to be set *before* the setter runs.